ibanizator.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'equalizer', '~> 0.0.11'
   s.add_dependency 'adamantium', '~> 0.2.0'
 
-  s.add_development_dependency 'rake', '~> 11.1.2'
-  s.add_development_dependency 'bundler', '~> 1.15.0'
-  s.add_development_dependency 'rspec', '~> 3.4'
+  s.add_development_dependency 'rake', '>= 11.1.2'
```

Comment:
any specific reason for using >= instead of ~> which ensures breaking updates ( when maximum number changes) will be excluded